### PR TITLE
add Docker and docker-compose

### PR DESCRIPTION
**Pull request by nicolabeghin**
## Motivation
Adding `Dockerfile` and `docker-compose.yml` for ready-to-use local build for all the not-frontend-proficient folks like me 😄 

## Verification Steps
    docker-compose build
    docker-compose up -d 

Access http://localhost:8080/adminv2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+RUN rm -fr /usr/local/bin/yarn* &&\
+    npm install -g yarn
+
+WORKDIR /app
+COPY . .
+
+RUN yarn
+
+CMD [ "yarn", "start" ]
```
Comment (edewit) [inline Dockerfile:11]: Wouldn't it be nicer to do a `yarn build` and then use a simple http server to serve the static files? That way the site would be a lot faster as this is developer mode, or is that what you wanted?

Comment (nicolabeghin) [reply Dockerfile:11]: @edewit totally agree, but tried to stick to "barebone" setup as per README for dev/sneak-peek mode (no PRD optimization at all).

**Comment by edewit** [comment]
@nicolabeghin would be good to add these instructions to build and run in the README

File: Dockerfile (hunk 1 above)
Comment (edewit) [inline Dockerfile:11]: how about this:
```suggestion
RUN yarn build

FROM nginx
EXPOSE 80
COPY --from=builder /app/build /usr/share/nginx/html
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM node:12
```
Comment (edewit) [inline Dockerfile:1]:
```suggestion
FROM node:12 as builder
```

**Comment by nicolabeghin** [comment]
@edewit updated with build and nginx, plus some basic performance optimizations

**Review by edewit — APPROVED** [review]

**Pull request merged**